functions/package-lock.json

```diff
@@ -0,0 +1,2388 @@
+{
```

Comment:
Should the lock file be this big?

Reply:
Yeh this is just from the default firebase and typescript modules. I guess they just have hella dependencies